hw/ip/entropy_src/data/entropy_src_testplan.hjson

```diff
@@ -141,9 +141,53 @@
             '''
     }
     {
-      name: seed_output_cg
+      name: seed_output_hw_cg
       desc: '''
-            Covers that data output is observed at the csrng, entropy_src, and fw_ov interfaces for
+            Covers that data output is observed at the CSRNG HW interface for
+            all possible modes of operation, including:
+            - CONF.FIPS_ENABLE (True and False)
+            - CONF.ENTROPY_DATA_REG_ENABLE (True and False)
+            - CONF.THRESHOLD_SCOPE (True and False)
+            - CONF.RNG_BIT_ENABLE (True and False)
+            - CONF.RNG_BIT_SEL (0 to 3)
+            - ENTROPY_CONTROL.ES_TYPE (True and False)
+            - FW_OV_MODE (True or False)
+            - FW_OV_ENTROPY_INSERT (True or False)
+            In addition to the above, the following settings are illegal when sampling on the
+            this covergroup, and merit the creation of illegal_bins
```

Comment:
Hi @matutem, 
   I'm not sure your previous suggestion quite fits in with the whole sentence, but what would you say to this?
```suggestion
            In addition to the above, the following settings are illegal when sampling  this
            covergroup, and merit the creation of illegal_bins.
```